corona cases forecasting/main.py

```diff
@@ -0,0 +1,320 @@
+        casesY.append(list1[i])
+        datesX.append((date.today()+ datetime.timedelta(days=i)).strftime('%m/%d/%Y'))
+
+    # Plot Results for forecasted dates only (detailed)
+
+    plt.plot(datesX,casesY,color='red')
+    plt.title('New active Cases Forecast')
+    plt.xticks(rotation=90)
+    # plt.figure(figsize=(22,22))
+    plt.savefig("plot1.png",bbox_inches='tight')
+    plt.autoscale()
+    plt.show()
+
+
+
+def death_cases():
+    #confirmed cases
+    india_daily_increase = daily_increase(india_deaths)
+
+    # Dates pre processing
+    days_since_1_22 = np.array([i for i in range(len(dates))]).reshape(-1, 1)
+
+    days_in_future = 0
+    future_forecast = np.array([i for i in range(len(dates)+days_in_future)]).reshape(-1, 1)
+
+    start = '1/22/2020'
+    start_date = datetime.datetime.strptime(start, '%m/%d/%Y')
+    future_forecast_dates = []
+    for i in range(len(future_forecast)):
+        future_forecast_dates.append((start_date + datetime.timedelta(days=i)).strftime('%m/%d/%Y'))
+
+    dataCovid= pd.DataFrame({ 'Dates': future_forecast_dates , 'Daily Increase':india_daily_increase })
+
+    train = dataCovid[:int(0.7*(len(dataCovid)))]
+    valid = dataCovid[int(0.7*(len(dataCovid))):]
+    #preprocessing (since arima takes univariate series as input)
+    train.drop('Dates',axis=1,inplace=True)
+    valid.drop('Dates',axis=1,inplace=True)
+
+    model = auto_arima(train, trace=True, error_action='ignore', suppress_warnings=True)
+    model.fit(train)
+
+    forecast = model.predict(n_periods=len(valid))
+    forecast = pd.DataFrame(forecast,index = valid.index,columns=['Prediction'])
+
+
+    def ARIMAmodel(series, order, days = 21):
+        # Fitting and forecast the series
+        train = [x for x in series]
+        model = ARIMA(train, order = order)
+        model_fit = model.fit(disp=0)
+        forecast, err, ci = model_fit.forecast(steps = days, alpha = 0.05)
+        start_day = date.today() + datetime.timedelta(days = 1)
+        predictions_df = pd.DataFrame({'Forecast':forecast.round()}, index=pd.date_range(start = start_day, periods=days, freq='D'))
+        return predictions_df, ci
+
+
+
+    new_deaths = dataCovid['Daily Increase'].values
+    order = {
+        'new_deaths': (0, 1, 1),
+
+    }
+    new_deaths_today=new_deaths[-1]
+    # Forecasting with ARIMA models
+    new_deaths_pred, new_deaths_ci = ARIMAmodel(new_deaths, order['new_deaths'])
+    casesY=[]
+    datesX=[]
+    list1 = new_deaths_pred.iloc[: ,0]
+    for i in range(0,21):
+        casesY.append(list1[i])
+        datesX.append((date.today()+ datetime.timedelta(days=i)).strftime('%m/%d/%Y'))
+
+    # Plot Results for forecasted dates only (detailed)
+
+    plt.plot(datesX,casesY,color='red')
+    plt.title('New death Cases Forecast')
+    plt.xticks(rotation=90)
+    # plt.figure(figsize=(22,22))
+    plt.savefig("plot2.png",bbox_inches='tight')
+    plt.autoscale()
+    plt.show()
+
+
+
+
+def recovered_cases():

```

Comment:
Here is the error, you previously defined the "recovered_cases" as a pandas data frame and now defining it as a function. This may not have any effect in Jupyter notebook as they are cell-based but in a Py script, it will give an error. Change this function name and at other instances where it is used